lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -28,6 +28,9 @@ def initialize(input, suspended_action:, id: nil)
         @passphrase = action_input['secrets']['key_passphrase']
         @execution_timeout_interval = action_input[:execution_timeout_interval]
         @cleanup_working_dirs = action_input.fetch(:cleanup_working_dirs, true)
+        @host = action_input.fetch(:hostname)
```

Comment:
In here we're dealing with multiple hosts at once so even if this is set, we should still probably do this for every host in the inventory.